www/index.md

```diff
@@ -0,0 +1,26 @@
+# Compensated
+Build products. Get paid.
+
+
+### Expand Your Sales Channels
+You started with a website, and now you have an iOS app.
+And oh yea, there’s that old Gumroad storefront you were experimenting with that has some recurring payments set up.
+
+No matter how you get (or got!) paid, Compensated standardizes the different payment processers data structures and event types so you can add sales channels without losing momentum on your customer facing features.
+
+### Prevent Vendor Lock-In
+Tired of your current payment providers fee structure?
+Sick of the weird refund policy that locks your account?
+
+Compensated helps you safely transition your customers to new payment processors without requiring system rewrites.
+
+### Read the Source, Luke
+All our source code is publicly available.
```

Comment:
I haven't really worked under this model for a paid product. Couldn't we just give access to the paying customers at first?

Reply:
An excellent question! I think we _could_, but it would mean we would want a source distribution mechanism other than "push to publicly readable git repository" and I don't have the time or expertise to set that up. 

I'd personally prefer to rely on existing copyright laws. Someone who can read a book from the library doesn't have the right to copy the book or re-sell the copies to their friends.  If someone violates the terms of the license, we can send them a cease and desist.

If they refuse, we could pursue further legal action commiserate with our economic ability to do so. Having a public record of source code contributions with timestamps gives us a public record of the intellectual property being owned by us and makes the case pretty clear.

Reply:
Yeah, managing the source release could get annoying. I guess the case I'm most worried about is people who might use this library or might decide to roll their own similar thing. If our source is out for anyone to see, people have a lot of sample code to help them understand how the systems work. I think the value of the product will probably be more in that understanding than in the exact code itself. If we want to put the source out there, I might prefer actual open source plus expert consulting that we focus on. But also, unfamiliar things are scarier so I could be worrying about a non-issue.

Reply:
I would love to license this as an OSI approved license once they pass something that allows us to not grant a license to organizations that violate human rights. 


But yes, I agree that "open source + hey if you want support + here's a hosted version" is the  economic model that would work best and is pretty familiar.